airbyte-integrations/connectors/source-stripe/source_stripe/manifest.yaml

```diff
@@ -0,0 +1,101 @@
+version: 6.11.1
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - charges
+
+definitions:
+  streams:
+    authorizations:
+      type: DeclarativeStream
+      name: authorizations
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: "{{ 'events' if timestamp(stream_slice['start_time']) > timestamp(day_delta(-30, format='%s')) else 'issuing/authorizations' }}"
+          http_method: GET
+          request_headers:
+            Stripe-Account: "{{config['account_id']}}"
+          request_parameters:
+            limit: "100"
+            types[]: "{{ ['issuing_authorization.created', 'issuing_authorization.request', 'issuing_authorization.updated'] if timestamp(stream_slice['start_time']) > timestamp(day_delta(-30, format='%s')) else None }}"
```

Comment:
This does not work as it will add the query param `types[]=None` which is not what we want. So there might be a need to add to the CDK the removal of query params where the value is None. This makes sense to me conceptually so we can probably just update [this line](https://github.com/airbytehq/airbyte-python-cdk/blob/57e1b524e2c55b2c1e8b80da38c0cfda3c6a385a/airbyte_cdk/sources/declarative/requesters/request_options/interpolated_request_options_provider.py#L24) to `ValidRequestTypes = (type(None), str, list)`